charts/stash-enterprise/templates/deployment.yaml

```diff
@@ -46,6 +46,13 @@ spec:
         - --v={{ .Values.logLevel }}
         - --docker-registry={{ .Values.operator.registry }}
         - --image-tag={{ .Values.operator.tag }}
+        {{ if .Values.imagePullSecrets }}
```

Comment:
Just use https://github.com/Masterminds/sprig/blob/master/docs/string_slice.md#join

{{ with .Values.imagePullSecrets }}
- --image-pull-secrets={{ join "," . }}
{{ end }}

Reply:
It will not work. `.Values.imagePullSecrets` is a map. Not slice.

Reply:
Right! We can get the values in a list using `values` https://github.com/Masterminds/sprig/blob/master/docs/dicts.md#values

`{{ values . | join "," }}`